src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -45,10 +45,11 @@ <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values base
       <a
         ng-href="{{field.details.visualizeUrl}}"
         ng-show="field.visualizable"
-        class="sidebar-item-button primary"
-        data-test-subj="fieldVisualize-{{::field.name}}">
-          Visualize
-          <span class="discover-field-vis-warning" ng-show="warnings.length" tooltip="{{warnings.join(' ')}}">
-          ( {{::warnings.length}} <ng-pluralize count="warnings.length" when="{'1':'warning', 'other':'warnings'}"></ng-pluralize> <i aria-hidden="true" class="fa fa-warning"></i> )
-          </span>
+        class="kuiButton kuiButton--primary kuiButton--small kuiButton--fullWidth"
+        data-test-subj="fieldVisualize-{{::field.name}}"
+      >
+        Visualize
```

Comment:
I kinda like the square corners in the first one, take it or leave it.
<img width="442" alt="screen shot 2017-04-21 at 10 19 38 am" src="https://cloud.githubusercontent.com/assets/16563603/25281690/186ed39c-267c-11e7-9eee-42267d4180ef.png">
<img width="272" alt="screen shot 2017-04-21 at 10 19 44 am" src="https://cloud.githubusercontent.com/assets/16563603/25281691/18700f3c-267c-11e7-8329-d8c9c39dd623.png">